ui/raidboss/data/03-hw/raid/a3s.js

```diff
@@ -203,8 +216,44 @@
       },
       infoText: {
         en: 'Embolus: Move Boss',
+        fr: 'Caillot: Déplacez le Boss',
```

Comment:
General remark. In French, you have to put a space before a punctuation mark than contains two parts (like : ; ? !)
So in that case it will be `fr: 'Caillot : Déplacez le boss',` (Ah oui et en français il n'y a pas de majuscule à chaque mot (sauf nom propre).

Reply:
Je corrige